.circleci/config.yml

```diff
@@ -145,22 +174,55 @@ commands:
               bundle exec fastlane update_carthage_commit
 
 jobs:
-  runtest:
-    <<: *base_job
+  run-test-ios:
+    <<: *base-job
+    steps:
+      - checkout
+      - install-dependencies
+      - run:
+          name: Run tests
+          command: bundle exec fastlane test_ios
+      - store_test_results:
+          path: fastlane/test_output
+      - store_artifacts:
+          path: fastlane/test_output/xctest
+          destination: scan-test-output
+
+  run-test-tvos:
+    <<: *base-job
     steps:
       - checkout
       - install-dependencies
       - run:
           name: Run tests
-          command: bundle exec fastlane test
+          command: bundle exec fastlane test_tvos
       - store_test_results:
           path: fastlane/test_output
       - store_artifacts:
           path: fastlane/test_output/xctest
           destination: scan-test-output
 
-  buildTvWatchAndMacOS:
-    <<: *base_job
+  run-test-ios-12:
+    <<: *base-job
+    steps:
+      - checkout
+      - install-dependencies
+      - install-and-create-sim:
+          install-name: iOS 12.4 Simulator
```

Comment:
Awesome!!
Is 12.4 the lowest or can we do 12.0?

Reply:
12.4 is the lowest 😔  Here is the full list 👇 

```sh
watchOS 5.3 Simulator (not installed)
iOS 12.4 Simulator (not installed)
tvOS 12.4 Simulator (not installed)
iOS 13.0 Simulator (not installed)
iOS 13.1 Simulator (not installed)
watchOS 6.0 Simulator (not installed)
tvOS 13.0 Simulator (not installed)
tvOS 13.2 Simulator (not installed)
iOS 13.2 Simulator (not installed)
iOS 13.3 Simulator (not installed)
watchOS 6.1.1 Simulator (not installed)
tvOS 13.3 Simulator (not installed)
iOS 13.4 Simulator (not installed)
iOS 13.5 Simulator (not installed)
tvOS 13.4 Simulator (not installed)
watchOS 6.2.1 Simulator (not installed)
iOS 13.6 Simulator (not installed)
iOS 13.7 Simulator (not installed)
iOS 14.0 Simulator (not installed)
watchOS 7.0 Simulator (not installed)
iOS 14.1 Simulator (not installed)
tvOS 14.0 Simulator (not installed)
iOS 14.2 Simulator (not installed)
watchOS 7.1 Simulator (not installed)
tvOS 14.2 Simulator (not installed)
tvOS 14.3 Simulator (not installed)
iOS 14.3 Simulator (not installed)
watchOS 7.2 Simulator (not installed)
iOS 14.4 Simulator (not installed)
tvOS 14.4 Simulator (not installed)
tvOS 14.5 Simulator (installed)
iOS 14.5 Simulator (installed)
watchOS 7.4 Simulator (installed)
iOS 15.0 Simulator (not installed)
watchOS 8.0 Simulator (not installed)
tvOS 15.0 Simulator (not installed)
```

Reply:
Sounds good, better than nothing 👍